tl/features/create_pseudo_gt.py

```diff
@@ -2,23 +2,80 @@
 from tl.exceptions import RequiredColumnMissingException
 from tl.file_formats_validator import FFV
 from tl.exceptions import UnsupportTypeError
+import numpy as np
+import sys
 
 
 def create_pseudo_gt(df: pd.DataFrame, column_thresholds: str,
-                     output_column: str):
+                     output_column: str, filter: str):
     column_thresholds = [_.split(":") for _ in column_thresholds.split(",")]
+
+    if filter:
+        col, val = filter.split(":")
+        val =  float(val)
+        if col not in df.columns:
+            raise RequiredColumnMissingException("The input column {} does not" 
+                                                 " exist in given data.".format(col))
     ffv = FFV()
     if ffv.is_candidates_file(df):
+        grouped = df.groupby(by=["column", "row"])
         for column, threshold in column_thresholds:
             if column not in df.columns:
```

Comment:
this is already checked at line 16, no need to check again. Also what the difference between column threshold and filter?

Reply:
In line 16, the column in filter is checked. Filter argument is to filter out the rows based on values. Columns in column threshold is used to make the pseudo gt.